### ISSUE TITLE
Can not build grpc_protoc_plugins.sln

### ISSUE THREAD

**Issue by goldenbull** [grpc/grpc#6011]
Seems an obsoleted dependency is not removed from sln file.

```
D:\git\grpc\vsprojects>msbuild grpc_protoc_plugins.sln /p:Configuration=Release
Microsoft (R) Build Engine version 14.0.24728.2
Copyright (C) Microsoft Corporation. All rights reserved.

Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
Build started 2016/3/30 9:11:41.
D:\git\grpc\vsprojects\grpc_protoc_plugins.sln : Solution file error MSB4051: Project {B6E81D84-2ACB-41B8-8781-493A944C
7817} is referencing a project with GUID {AAC6AF12-94C8-4A3C-A1BF-DAA4738F4500}, but a project with this GUID was not f
ound in the .SLN file.

Build FAILED.

  D:\git\grpc\vsprojects\grpc_protoc_plugins.sln : Solution file error MSB4051: Project {B6E81D84-2ACB-41B8-8781-493A94
4C7817} is referencing a project with GUID {AAC6AF12-94C8-4A3C-A1BF-DAA4738F4500}, but a project with this GUID was not
 found in the .SLN file.

    0 Warning(s)
    1 Error(s)

Time Elapsed 00:00:00.01
```

**Comment by jtattermusch** [comment]
This also breaks our artifact build:

https://grpc-testing.appspot.com/view/Artifacts/job/gRPC_build_artifacts/architecture=x86,language=protoc,platform=windows/527/console

**Comment by jtattermusch** [comment]
Very likely related to: https://github.com/grpc/grpc/pull/5951

**Comment by jtattermusch** [comment]
Definitely caused by #5951.
Here's the missing project (it was in protoc target, but now it's in all target, which moved it from grpc_protoc_plugins.sln to grpc.sln):
`"grpc++_codegen_lib", "vcxproj\.\grpc++_codegen_lib\grpc++_codegen_lib.vcxproj", "{AAC6AF12-94C8-4A3C-A1BF-DAA4738F4500}"`

**Comment by ctiller** [comment]
Fixed in commit 03d8f2f

**Comment by jtattermusch** [comment]
@ctiller  Thanks!
